### PR TITLE
Improve Notebooks/Replay documentation

### PR DESCRIPTION
**Pull request by enolfc**
# Summary

Updates documentation, getting ready for Notebooks updates and Replay service.
Should be merged as soon as the changes are approved in SSB/CHM

Adds
- RStudio
- CVMFS
- nb_conda_kernels
- Replay (instead of Binder)

---

**Related issue :**

### CODE REVIEW

**Review by sebastian-luna-valero — APPROVED** [review]
Thanks!

**Comment by gwarf** [comment]
@enolfc is this PR waiting as draft on something specific?

File: content/en/users/dev-env/replay/_index.md [hunk 1]
```diff
@@ -118,3 +119,16 @@ those spaces configured to be mounted locally will be made available
...
+## Errors when launching
+
+You can only have one notebook server instance running at any given time, if you
+try to launch a second instance while there is already one running you will get
+an error like shown in the screenshot:
+
+![Launch Error](launch-error.png)
+
+In that case, you can either stop your existing server or connect to it from your
+[Replay JupyterHub home](https://replay.notebooks.egi.eu/hub/hub/home):
```
Comment (sebastian-luna-valero) [inline content/en/users/dev-env/replay/_index.md:132]: Is the double `/hub/hub` correct?

```suggestion
[Replay JupyterHub home](https://replay.notebooks.egi.eu/hub/home):
```

**Review by sebastian-luna-valero — APPROVED** [review]
Minor comments below, thanks!

**Review by gwarf — APPROVED** [review]
LGTM, once all suggestions will have been addressed and the service will be in a good shape (certificate and so on)

File: content/en/users/dev-env/replay/_index.md (hunk 1 above)
Comment (enolfc) [reply content/en/users/dev-env/replay/_index.md:132]: True!

**Review by gwarf — CHANGES_REQUESTED** [review]
The certificate for the service is not valid and should be replaced before this PR can be merged.

**Comment by enolfc** [comment]
> The certificate for the service is not valid and should be replaced before this PR can be merged.

replay hostname now in place. binder redirects to replay also.

File: content/en/users/dev-env/replay/_index.md [hunk 2]
```diff
@@ -86,17 +87,17 @@ OS packages; and even complete custom Dockerfiles to bring any application to
...
 for creating your own reproducible environment. To run this directly on EGI's
-Binder click on the button below:
+Replay click on the button below:
 
-[![Binder](https://binder.notebooks.egi.eu/badge_logo.svg)](https://binder.notebooks.egi.eu/v2/gh/EGI-Federation/binder-example/HEAD)
+[![Binder](https://replay.notebooks.egi.eu/badge_logo.svg)](https://replay.notebooks.egi.eu/v2/gh/EGI-Federation/binder-example/HEAD)
 
-You can create such link to share your notebooks from the Binder interface, as
+You can create such link to share your notebooks from the Replay interface, as
 shown in the screenshot below, you can copy the URL shown when the building is
```
[the remark this answers is not in the document]
Comment (gwarf) [reply content/en/users/dev-env/replay/_index.md:101]: @enolfc, is this something you want to address before merging, or should we merge like this?

**Review by gwarf — APPROVED** [review]

File: content/en/users/dev-env/replay/_index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (enolfc) [reply content/en/users/dev-env/replay/_index.md:101]: Image is now updated

**Pull request merged**